src/main/java/uk/ac/cam/cl/dtg/isaac/api/Constants.java

```diff
@@ -45,6 +47,8 @@
      * Game specific variables.
      */
     public static final int GAME_BOARD_TARGET_SIZE = 10;
+    public static final Set<String> FASTTRACK_GAMEBOARD_WHITELIST
+            = ImmutableSet.of("ft_core_2017"); // Keep in sync with APP's fastTrackProgressEnabledBoards constant
```

Comment:
It would be nice to avoid hard coding gameboard ids in the constants file... If you think it is necessary, can we make it a config value so we don't have to rebuild the api if we want to add more fast track boards?

Reply:
Sure, it would be nice to be able to update the list without needing a release. I will do that.